Library/Styles/BaseStyles.swift

```diff
@@ -4,6 +4,7 @@ import UIKit
 
 public enum Styles {
   public static let cornerRadius: CGFloat = 4.0
+  public static let buttonMinHeight: CGFloat = 44.0
```

Comment:
This value seems like such a generic constant (based on HIG) not really related to just buttons..how about calling it something like `minTouchDimensions` or `minTappableDimensions`..etc?

Reply:
Updated to `minTouchSize: CGSize`